.github/workflows/prepare_post_release.yml

```diff
@@ -51,6 +51,12 @@ jobs:
             ${{ github.token }} \
             ${{ github.repository }} \
             ${{ steps.pipeline_context.outputs.candidate_tag }}
+      - name: "Create backport label"
+        run: |
```

Comment:
I think you'll need to add the `GITHUB_TOKEN` env variable. See https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows.

Reply:
Thanks so much for the review @waiting-for-dev 🙌 

I added the token and tested all the flow in a personal fork:
- [Prepare Release PR](https://github.com/rainerdema/solidus/pull/2)
- ["Create backport label"](https://github.com/rainerdema/solidus/actions/runs/4788068474/jobs/8514159963#step:10:1) job
- [List of labels ](https://github.com/rainerdema/solidus/labels)with the new one correctly created

After your comment, I realized that the ["Check for the presence of a changelog label"](https://github.com/solidusio/solidus/blob/019ba27e24fcbc255db7fd3efdb5820efe0662c7/.github/workflows/ensure_changelog_label.yml#L13-L19) job has the same problem:
- [Example](https://github.com/solidusio/solidus/actions/runs/4786766393/jobs/8511113868#step:3:9)
![Screenshot 2023-04-24 at 17 41 53](https://user-images.githubusercontent.com/19948291/234047418-ae00f8c9-3f47-4668-8b11-4a6e3313302a.png)


I will open a separate PR to address this problem on that job as well 👍